### ISSUE TITLE
Compatibility with Full Keyboard Access

### ISSUE THREAD

**Issue by djsavvy** [blinksh/blink#988]
Currently, Blink does not work in a usable state with the ‘Full Keyboard Access’ setting turned on in ‘Settings > Accessibility > Keyboards > Hardware Keyboards.’ 

For instance, see #987. 

What would be required for Blink to be compatible with Full Keyboard Access?

**Comment by yury** [comment]
Go to Settings.app -> Accessibility -> Keyboards -> Full Keyboard Access -> Commands and remap `Active` to SHIFT+space for instance, instead of just space.

I’m closing this issue, feel free to reopen if needed.
